components/landing/LandingCta.vue

```diff
@@ -75,5 +78,16 @@ export default class extends Vue {
     margin-left: $spacing-06;
     fill: currentColor;
   }
+
+  svg {
+      overflow: visible;
+  }
+  path:nth-child(1) {
+    transform: translate(0, 0);
+    transition: transform 0.2s ease-in-out;
+  }
+  &:hover path:nth-child(1) {
```

Comment:
Yes, both are applying.

You can check it on Preview link that the Vercel bot commented:  https://qiskitorg-git-fork-korgan00-feature-gi-i909-link-ar-b7dec5.qiskit.vercel.app

Reply:
I think they make sense for `hover` and `focus`, but not active. Looks like we currently aren't doing anything for Active? Maybe we should make a separate issue to think about what should happen on `active`. It could just be a color change to purple 90.

Reply:
Actually, it looks like Salva has some thoughts on the active state over on https://github.com/Qiskit/qiskit.org/pull/908#issuecomment-692932242 regarding active for mobile. I trust his thoughts on this one.